src/main/java/com/chikli/hudson/plugin/naginator/NaginatorBuildTriggerBadgeDeactivator.java

```diff
@@ -0,0 +1,14 @@
+package com.chikli.hudson.plugin.naginator;
+
+import hudson.Extension;
+import hudson.model.Cause;
+import org.jenkinsci.plugins.buildtriggerbadge.provider.BuildTriggerBadgeDeactivator;
+
+@Extension
```

Comment:
Add `optional=true` as buildtriggerbadge is optional.
(Reference: https://javadoc.jenkins.io/hudson/Extension.html#optional() )

Reply:
done